dali/test/python/test_fw_iterators.py

```diff
@@ -1003,6 +1003,27 @@ def stop_teration_case_generator():
                     for infinite in [False, True]:
                         yield batch_size, epochs, iter_num, auto_reset, infinite
 
+def check_iterator_wrapper_first_iteration(BaseIterator, *args, **kwargs):
+    class IteratorWrapper(BaseIterator):
+        def __init__(self, *args, **kwargs):
+            self._allow_next = False
+            super(IteratorWrapper, self).__init__(*args, **kwargs)
+
+        def __next__(self):
+            assert(self._allow_next)
+            outs = super(IteratorWrapper, self).__next__()
+
+    pipe = Pipeline(batch_size = 16, num_threads = 1, device_id = 0)
+    with pipe:
+        data = fn.uniform(range=(-1, 1), shape=(20, 30, 3), seed=1234)
```

Comment:
```suggestion
        data = fn.uniform(range=(-1, 1), shape=(2, 2, 2), seed=1234)
```
I think we can make it as simple as possible